pact/coin-contract/coin.pact

```diff
@@ -57,7 +72,7 @@
     \the transaction, and the price is the spot price of gas at that time.    \
     \The gas buy will be executed prior to executing SENDER's code."
 
-    @model [(property (> total 0.0))]
+    (enforce-unit total)
```

Comment:
Is the minimum defined here applied to all Pact numbers? Say if I compose `transfer` off some math calculation, like in `(transfer "foo" "bar" (/ numer denom))` and the result of the division is something funky and long. Would we expect the `transfer` to fail or to automatically truncate?

Reply:
I think we could extract a utility function to do the truncate to help smart contract math.